### ISSUE TITLE
SSD1963 interface

### ISSUE THREAD

**Issue by RushHour99** [t0mac0/working#1]
@t0mac0 I'm not sure if it's possible to get in touch with you regarding a SSD1963 question for configuring a tft screen...

I have a 5.2" bar tft screen that I am trying to configure with an 8-bit interface using the fbtft driver.  I don't have great documentation from the manufacturer to initialize the screen, but I have made some progress.

I got to a point where my screen is showing what looks to be 3 copies of an image superimposed on itself.  It looks very similar to your screenshot from Feb 24th 2014 in this issue:

https://github.com/notro/fbtft/issues/58

You seem to reach a conclusion on Feb 26th 2014 that the fbtft driver is not sending the image in 3 8-bit cycles for RGB when configured for an 8-bit interface.  Maybe the fbtft driver is sending 2-bytes in RGB565 format and this explains why the image is incorrectly wrapping on itself since the tft controller is expecting 3 bytes.

Did you ever reach a conclusion if the fbtft will work with buswidth=8?  The other mentions of a working SSD1963 tft seem to be using a 16-bit buswidth.

Thanks!

**Comment by t0mac0** [comment]
yes, after many tests, I achieve to works, but, the frame rate is too slow, arround 5 frames per seconds, and the kernel spends many resources handling the TFT, but remenber that I used the raspberry pi 1, 512 Mb, 700 Mhz 1 core, I think that you can drive the LCD with 16 bits bus, much better with raspberry pi 3 B+. I don't remeber where is that code, it was many years ago.
